### DISCUSSION TITLE
How to make sure keyboard entry is completed

### DISCUSSION THREAD

**Discussion by lwahonen** [nut-tree/nut.js#382] [Q&A]
I'm trying to use nut.js to input text strings via the keyboard.

Entering "AAA", "BBB", "CCC" results in the control getting "AABCABBCC" or something similar. How to ensure the entry has been completed, before starting the next one?

**Comment by s1hofmann** [comment]
Hi @lwahonen 

could you please provide a short piece of sample code?

**Reply by s1hofmann** [reply]
@lwahonen Any update?
